backend/routes/content.py

```
@@ -391,7 +437,98 @@ def mark_module_complete_and_open_next(module_id, user_id):
+                    "moduleType": solution_module.module_type.name,
+                }
+            ),
+            200,
+        )
+    except Exception as e:
+        logger.error(
+            f"Error identifying challenge solution module for module {module_id}: {str(e)}"
+        )
+        return jsonify({"error": str(e)}), 500
+
+def get_num_modules_completed(user_id, unit_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_num_modules_completed`

(ANN201)